### DISCUSSION TITLE
Settings getting reset

### DISCUSSION THREAD

**Discussion by FireStorm005** [dhowe/AdNauseam#1850] [Q&A]
The setting for "Don't hide non-tracking Ads" keeps getting reset frequently, I haven't checked to be sure but it might be as often as every time I open the browser.  I'm on Firefox 88.0.1 64bit with AdNauseam v3.10.4.  Once I recheck the box it stays for that browser session.  Could this be a browser issue or a problem with AdNauseam?  Is this setting stored as a cookie as I do have Firefox set to block all cross-site cookies?

**Comment by dhowe** [comment]
I cannot confirm this on my system with the same version of FF.
Can you try with a fresh Firefox profile, as described [here](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles) ?

**Reply by FireStorm005** [reply]
It did not present the same behavior with a fresh profile, and rereading my original post i realized I got it backwards and I want to uncheck the "don't hide non-tracking Ads" and it keeps re-checking the box.

**Comment by dhowe** [comment]
Am I correct that the fresh profile fixed the problem, or that it persists?

**Reply by FireStorm005** [reply]
Yes, a fresh profile did fix it, does this mean I'll have to clear and re-setup my browser (extensions, bookmarks, etc) or is there another possible solution?

**Comment by dhowe** [comment]
Once a Firefox profile becomes corrupted like this, the best thing is to switch to another one. But you should be able to export your bookmarks and settings.
